### PR TITLE
Added about Page

### PR DESCRIPTION
**Pull request by RamGoel**
Added About Page as Given in the Issue (#29 )

[screen-capture (3).webm](https://user-images.githubusercontent.com/71088429/185781733-083cee85-d6c3-4bfe-84fb-8bdfb4a0412d.webm)

### CODE REVIEW

**Comment by pulkit-30** [comment]
Great 🎉 
Also, can we make it responsive for smaller devices?

File: stylesheets/aboutPage.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
```
Comment (pulkit-30) [inline stylesheets/aboutPage.css:1]: also, import this in HTML files

File: stylesheets/aboutPage.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    line-height: 35px;
+    text-align: justify;
+}
+
+#about-content div{
+    width: 90%;
+}
+#about-content {
+    width: 90%;
+    font-family: Poppins
```
Comment (pulkit-30) [inline stylesheets/aboutPage.css:13]: We can add this as a `root` variable as `--ternary-font: Poppins`

**Comment by RamGoel** [comment]
> Great 🎉 Also, can we make it responsive for smaller devices?

The issue (#23) is assigned to other contributors. If still about page doesn't seem right on mobile devices. I'll fix it.

**Comment by pulkit-30** [comment]
You can fix it 🚀

**Comment by RamGoel** [comment]
> You can fix it 🚀

@darshikaguptaa is working on makin it responsive.  @pulkit-30 Approve this PR so that she can pull the About Page Also.

**Review by pulkit-30 — APPROVED** [review]
LGTM 🎉 ,
the responsive issue can be fixed later 

not sure about the constellation png image and the about content.
cc: @dewank07 @Anv3sh

**Pull request merged**